Documenting Horovod code (#496)
flyteorg/flytesnacks: cookbook/case_studies/ml_training/spark_horovod/README.rst

.. _spark_horovod:

> Forecast Sales Using Rossmann Store Sales Data With Horovod and Spark

Reviewer: ```suggestion
Forecasting Rossman Store Sales with Horovod and Spark
```

(the "with" is a preposition, and so should be lower case)

Reviewer 2: It depends. In [APA](https://apastyle.apa.org/style-grammar-guidelines/capitalization/title-case) style title case, "with" has to be "With", whereas in Chicago style title case, "with" has to be "with".

    :alt: Flyte-Spark plugin

    Flyte-Spark plugin

> To set-up the plugin during Flyte deployment, refer to `this. <https://github.com/flyteorg/flytesnacks/blob/master/cookbook/docs/integrations.rst>` __

Reviewer 2: ```suggestion
The plugin needs some configuration to be done in the Flyte backend as well. Refer to :ref:`Kubernetes Spark Jobs <plugins-spark-k8s>` for the installation instructions.
```

> The plugin needs some configuration to be done in the Flyte backend as well. Refer to :ref:`Kubernetes Spark Jobs <plugins-spark-k8s>` for the installation instructions.

Reviewer: ```suggestion
The plugin requires configuring the Flyte backend as well. Refer to :ref:`Kubernetes Spark Jobs <plugins-spark-k8s>` for setup instructions.
```
